notebooks/official/migration/sdk-custom-image-classification-custom-container.ipynb

```diff
@@ -78,23 +80,23 @@
         "\n",
         "This tutorial uses the following Google Cloud ML services and resources:\n",
         "\n",
-        "- *Vertex AI Training*\n",
-        "- *Vertex AI Model Registry*\n",
-        "- *Vertex AI Batch Predictions*\n",
-        "- *Vertex AI Endpoints*\n",
+        "- `Vertex AI Training`\n",
+        "- `Vertex AI Model Registry`\n",
+        "- `Vertex AI Batch Predictions`\n",
```

Comment:
remove back tics and change to lower case. Should be:
- Vertex AI batch predictions